daemon/cluster/executor/container/container.go

```diff
@@ -124,7 +125,6 @@ func (c *containerConfig) config() *enginecontainer.Config {
 		Env:        c.spec().Env,
 		WorkingDir: c.spec().Dir,
 		Image:      c.image(),
-		Volumes:    c.volumes(),
```

Comment:
Changes in this `daemon/cluster/executor/container/container.go` is from https://github.com/docker/swarmkit/pull/1563